plugins/polio/js/src/domains/VaccineModule/StockManagement/StockVariation/Modals/validation.ts

```diff
@@ -44,7 +44,7 @@ export const useFormAValidation = () => {
         lot_numbers: yup
```

Comment:
Why not comment the whole lot_numbers validation as it's not used for now? But not blocking